### ISSUE TITLE
ZWave Controller upgrade

### ISSUE THREAD

**Issue by madansu** [zwave-js/zwave-js-ui#3330]
Hi Everyone,

I have a Aeotec ZWave stick (gen 5) that is connected to ZWaveJS. 

I would like to look at changing the stick to the series 800 stick from Zooz (https://www.amazon.com/Z-Wave-ZST39-Assistant-HomeSeer-Software/dp/B0BW171KP3/ref=sr_1_1?crid=2BDKIZV7XVBJJ&keywords=zooz%2B800&qid=1696810575&sprefix=zooz%2B%2Caps%2C137&sr=8-1&th=1)

To perform this change, do i simply unplug the old stick and plug in the new stick ?

Would the nodes reconnect to the new controller once the controller is up and ready ?

I am running ZWaveJS in a docker container on a Linux host in ESXi 8

The steps i plan on following are - 
1. shut down the ZWaveJS docker container
2. shut down the host running docker
3. unplug the Aeotec stick
4. plug in the Zooz 800 series ZWave stick
5. Pass the Zooz stick to the docker host as a USB device
6. Restart the docker host
7. Restart the ZwaveJS docker container

Here is my config for the zwavejs container 

  zwavejs:
    container_name: zwavejs
    image: zwavejs/zwavejs2mqtt:latest
    restart: always
    tty: true
    stop_signal: SIGINT
    environment:
      - SESSION_SECRET=**REDACTED**
    devices:
      - '/dev/ttyACM0:/dev/ttyACM0'
    volumes:
      - /data/docker/volumes/zwavejs/store:/usr/src/app/store
    ports:
      - '8091:8091' # port for web interface
      - '3000:3000' # port for zwave-js websocket server

Please let me know if this would work ?

**Comment by robertsLando** [comment]
In order to do this you should firstly do an NVM backup of your stick and restore that backup on the new one once you plug it. 

@AlCalzone Is the Aeotec gen 5 NVM backup compatible with the zooz 800?

**Comment by AlCalzone** [comment]
Your only way to migrate to the 800 series right now is the controller shift, which requires external tools and is [explained here](https://community.home-assistant.io/t/take-over-zwave-network/490219/4).

**Comment by madansu** [comment]
@robertsLando and @AlCalzone 
Thank you for the information. 
I decided to stick within the manufacturer family and ordered a ZStick 7 from aeotec. 
I did see the controller migration steps and i'll report back once the migration is complete. 


Also, when upgrading to the latest version of zwavejs2mqtt, i have an issue where the controller never loads, and I keep getting the error "zwavejs driver is not ready yet, statistics will be enabled on driver initialization" and the list of devices never loads. I had to hard pin the version to 8.26 

Any idea what could be causing the issue ? My ZStick Gen 5 is at firmware 1.1

**Comment by robertsLando** [comment]
Check this: https://github.com/zwave-js/node-zwave-js/issues/6341

TLDR: Upgrade controller firmware or simply disable soft reset from settings

**Comment by AlCalzone** [comment]
The latest update also increases a timeout that's necessary for some Gen5 sticks.

**Comment by madansu** [comment]
Thank you .. I should have a new ZSTICK7 arriving in the next couple of days and I can give these steps a try. 
Also, I am on ESXi 8.0U1.

**Comment by AlCalzone** [comment]
700 series doesn't have the issues that surfaced recently. I recommend not upgrading past 7.18.x at the moment though.

**Comment by madansu** [comment]
Thank you very much for your help. As of this afternoon, I have upgraded my ZSTICK GEN 5 to FW 1.2 and disabled soft reset and the controller is working as before! 
In the upcoming days I will look to replace the Gen 5 stick with the Gen 7 stick - once I get around to doing a NVM backup and restore
